app/scripts/controllers/login.js

```diff
@@ -8,22 +8,25 @@
  * Controller of the mockuperApp
  */
 angular.module('mockuperApp')
-    .controller('LoginCtrl', ['$rootScope', '$scope', '$location', 'loginService', '$cookieStore', 
-      function($rootScope, $scope, $location, loginService, $cookieStore) {
+    .controller('LoginCtrl', ['$rootScope', '$scope', '$location', 'loginService', '$cookieStore', 'projectService',
+      function($rootScope, $scope, $location, loginService, $cookieStore, projectService) {
         $scope.userName = '';
+        $scope.identifier = '';
         $scope.password = '';
         $scope.userInfo = {};
         $scope.singIn = function() {
             var identifier = {
-                "username": $scope.userName
+                "email": $scope.identifier,
+                "password": $scope.password
             };
-            loginService.loginLogLogin.save(identifier).$promise.then(function(result) {
+            loginService.loginUser.save(identifier).$promise.then(function(result) {
                 console.log(result);
+                console.log("LoginCtrl");
```

Comment:
line 23,24 should be removed, I know they are useful at the development stage in master branch we should have cleaner code